lib/globus/client.rb

```diff
@@ -5,6 +5,7 @@
 require "ostruct"
 require "singleton"
 require "zeitwerk"
+require "byebug"
```

Comment:
This is only a local dev dependency, so I think we should consider moving this into `spec/spec_helper.rb` like we do in e.g. dor-services-client, sdr-client.

Reply:
mistakenly committed, and thus removed (it already exists in spec_helper)